01-html/04-nytimes.html

```diff
@@ -1,3 +1,1120 @@
+        </ul>
+        <li><a href="http://www.nytimes.com/crosswords/index.html?page=home&_r=0">Crossword</a></li>
+        <li><a href="http://www.nytimes.com/section/insider">Times Insider</a></li>
+        <li><a href="http://www.nytimes.com/pages/multimedia/index.html">Multimedia</a></li>
+        <li><a href="http://lens.blogs.nytimes.com/">Photography</a></li>
+        <li><a href="https://www.nytimes.com/store/">NYT Store</a></li>
+        <li><a href="http://www.nytwineclub.com/">NYT Wine Club</a></li>
+        <li><a href="https://nytedu.com/">nytEducation</a></li>
+        <li><a href="">More</a></li>
+        <ul>
+          <li><a href="http://www.nytimes.com/times-journeys/">Times Journeys</a></li>
+          <li><a href="http://www.nytimes.com/subscriptions/">Subscribe</a></li>
+          <li><a href="https://myaccount.nytimes.com/">Manage Account</a></li>
+          <li><a href="http://www.nytimes.com/pages/todayspaper/index.html">Today's Paper</a></li>
+          <li><a href="http://www.nytimes.com/interactive/blogs/directory.html">Blogs</a></li>
+          <li><a href="http://www.nytimes.com/pages/topics/">Times Topics</a></li>
+          <li><a href="http://www.nytimes.com/marketing/tools-and-services/">Tools & Services</a></li>
+          <li><a href="http://www.nytimes.com/section/jobs">Jobs</a></li>
+          <li><a href="http://www.nytimes.com/ref/classifieds/">Classifieds</a></li>
+          <li><a href="http://www.nytimes.com/pages/corrections/index.html">Corrections</a></li>
+        </ul>
+    </nav>
+    <div id="search" class="inline-block">
+      <img src="http://uploads.webflow.com/55cf187a8cacc9ab28d524b9/55d243680a3ebde1455ffab4_transparency.png" height="30" alt="search-icon">SEARCH</div>
+    <button class="inline-block"><a href="http://www.nytimes.com/subscriptions/Multiproduct/lp3004.html?module=Subscribe-Now&pgtype=Homepage">SUBSCRIBE NOW</a></button>
+    <button class="inline-block"><a href="http://www.nytimes.com/?action=Click&region=TopBar&module=EditionToggleToUS&pgtype=Homepage&WT.z_jog=1&hF=t&vS=undefined">LOG IN</a></button>
+    <nav id="gear-menu">
+      <img src="http://simpleicon.com/wp-content/uploads/gear-21.svg" alt="menu-gear" height="20">
+      <ul class="no-text-decoration">EDITION
+        <li><a href="http://www.nytimes.com/section/us">U.S.</a></li>
+        <li><a href="http://international.nytimes.com/?WT.z_jog=1&hF=t&vS=undefined">International</a></li>
+        <li><a href="http://cn.nytimes.com/zh-hant/"><img src="http://virtusjustitia.vjcdn.com/wp-content/uploads//2014/04/20121025032309-89444.jpg" height="15" alt="mallet-stickfigure">(Chinese)</a>
+        </li>
+      </ul>
+      <ul class="no-text-decoration">HELP
+        <li><a href="http://www.nytimes.com/content/help/front.html">FAQ</a></li>
+        <li><a href="http://www.nytimes.com/content/help/contact/directory.html">Contact Us</a></li>
+      </ul>
+    </nav>
```

Comment:
It might help you to begin a layout like this by drawing boxes on a piece of paper. Think about organization and layout. If I were approaching this page, everything from "Sections" to the Gear menu would be in one horizontal box. The logo, sub-header, and ads would be a separate horizontal box, and the nav bar would be a third horizontal box. All of these would be contained within a single `header` element.

```
<header>
  <div id="top-nav">
     ...sections, search, subscribe, log in, gear
  </div>
  <div id="branding">
    <div id="edition-selector">
       ...US, International, etc
    </div>
    <div class="ad-branding" id="ad-branding-left"></div>
    <div id="branding-logo">
       logo, dates, etc
    </div>
    <div class="ad-branding" id="ad-branding-right"></div>
  </div>
  <nav>
     world, us, politics, etc
  </nav>
</header>
<main>
...
```